src/cloud-api-adaptor/podvm/files/etc/systemd/system/kata-agent.service

```diff
@@ -2,7 +2,7 @@
 Description=Kata Agent
 BindsTo=netns@podns.service
 Wants=process-user-data.service
-After=netns@podns.service process-user-data.service
+After=netns@podns.service cloud-final.service process-user-data.service
```

Comment:
`cloud-final.service` -> `cloud-init.service`

Reply:
Thanks @liudalibj looks we can drop the cloud-final.service in kata-agent as the kata-agent config is static now.